.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: tests
+on: push
+jobs:
+  test:
+    name: Test (Ruby ${{ matrix.ruby }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
+        ruby: [ '2.3', '2.4', '2.5', '2.6', '2.7' ]
```

Comment:
Does `event_sourcery` work with Ruby 3.0 released recently?

Reply:
I hope so. Let's explore in another PR.